src/Locator.h

```diff
@@ -44,5 +45,6 @@ struct Locator
 	using livingActionSystem = entt::locator<ecs::systems::LivingActionSystemInterface>;
 	using townSystem = entt::locator<ecs::systems::TownSystemInterface>;
 	using pathfindingSystem = entt::locator<ecs::systems::PathfindingSystemInterface>;
+	using terrainSystem = entt::locator<LandIsland>;
```

Comment:
This is not an interface like the other systems. Please have a look at the others to see how they're done.

We're going to want an abstract class interface that Landisland implements and which allows us to easily swap out for another interface in the future.

Reply:
Fixed on the next commit